README.md

```diff
@@ -15,7 +15,7 @@
 
 ### 1. Import the registry via the VRChat Creator Companion (VCC)
 
-**[Add to VCC](vcc://vpm/addRepo?url=https%3A%2F%2Fkurone-kito.github.io%2Fvpm%2Findex.json)**
+Visit the **[VPM Catalogue page](https://kurone-kito.github.io/vpm/)** and click on the **Add to VCC** button.
 
 ### 2. Import the package to your project
 
```

Comment:
Your correction is inappropriate. The point you are making should be expressed more succinctly.

Reply:
Your proposal is concise enough. I want to incorporate it, but since this pull request already merged, I'll create a different pull request and correct it there.